### PR TITLE
Adding example for quickstart

### PR DESCRIPTION
**Pull request by akalburgi**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/tools/visual.rst [hunk 1]
```diff
@@ -9,4 +9,14 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``
+
+
+Visual for Quickstart
+*********************
+1. Generate the dar ``daml build``
```
Comment (neil-da) [inline docs/source/tools/visual.rst]:
```suggestion
1. Generate the dar using ``daml build``
```

File: docs/source/tools/visual.rst [hunk 2]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
+
+
+Visual for Quickstart
+*********************
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: I think you'll need a newline after this title.
```suggestion
*********************

```

File: docs/source/tools/visual.rst [hunk 3]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
+
+
+Visual for Quickstart
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: I'd change this to something more descriptive, eg
```suggestion
Example: visualizing the Quickstart project
```

You'll need to update the line beneath so the title underline is long enough.

File: docs/source/tools/visual.rst [hunk 4]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
+*********************
+1. Generate the dar using ``daml build``.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: Can you add a sentence or two of introduction here, explaining what this is? Essentially saying if you've run through the quickstart guide (or even potentially just cloned it, that might do), you can generate a visual like this. And don't forget to link to the quickstart guide (using `:doc:` - you can search through the docs folder to see examples of how that works)

Comment (bethaitman) [reply docs/source/tools/visual.rst]: You also have an assumption that they've installed Graphviz, might want to be explicit about that.

File: docs/source/tools/visual.rst [hunk 5]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
+2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``.
+3. Generate image ``dot -Tpng quickstart.dot quickstart.png``.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]:
```suggestion
3. Generate the visual graph with Graphviz by running ``dot -Tpng quickstart.dot quickstart.png``
```

File: docs/source/tools/visual.rst [hunk 6]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
+
+
+Visual for Quickstart
+*********************
+1. Generate the dar using ``daml build``.
+2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: Avoid full stops after code to run like this, people often copy them by accident
```suggestion
2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``
```

File: docs/source/tools/visual.rst (hunk 4 above)
Comment (bethaitman) [inline docs/source/tools/visual.rst]:
```suggestion
1. Generate the dar using ``daml build``
```

File: docs/source/tools/visual.rst [hunk 7]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: Avoid full stops after code to run like this, people often copy them by accident
```suggestion
4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``
```

File: docs/source/tools/visual.rst [hunk 8]
```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
+
+
+Visual for Quickstart
+*********************
+1. Generate the dar using ``daml build``.
+2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``.
+3. Generate image ``dot -Tpng quickstart.dot quickstart.png``.
+
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: Can you add a line here to introduce the image? Just to say that running the above should produce something like this.

File: docs/source/tools/visual.rst [hunk 9]
```diff
@@ -9,4 +9,21 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``
+
+
+Example: Visualizing the Quickstart project
+*******************************************
+
+If you are working through :doc:`quickstart </getting-started/quickstart>` example you can generate a visual like this. If you do not have Graphviz installed `install it from <http://www.graphviz.org/download/>`_.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]:
```suggestion
Here's an example visualization based on the :doc:`quickstart </getting-started/quickstart>`. You'll need to  `install Graphviz <http://www.graphviz.org/download/>`_ to try this out.
```

File: docs/source/tools/visual.rst [hunk 10]
```diff
@@ -9,4 +9,21 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
...
+
+1. Generate the dar using ``daml build``
+2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``
+3. Generate the visual graph with Graphviz by running ``dot -Tpng quickstart.dot quickstart.png``
+
+Running the above should produce an image which looks something like.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]:
```suggestion
Running the above should produce an image which looks something like this:
```

File: docs/source/tools/visual.rst (hunk 4 above)
Comment (akalburgi) [reply docs/source/tools/visual.rst]: Cool thanks for the suggestion. Did not notice how often I myself do that.

**Comment by akalburgi** [comment]
> Thanks for making the changes, now looks good to me!

Thank you so much

**Pull request merged**